File: go-sdk/pkg/models/operations/getuserv2.go
package operations

import (
	"openapi/pkg/models/shared"
)

type GetUserV2PathParams struct {
	ID string `pathParam:"style=simple,explode=false,name=id"`
}

type GetUserV2Request struct {
	PathParams GetUserV2PathParams
}

type GetUserV2Response struct {
	ContentType string
	StatusCode  int64
	UserV2      *shared.UserV2
}

File: user-api/ts-sdk/src/sdk/models/operations/getuserv2.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetUserV2PathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: string;
}


export class GetUserV2Request extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetUserV2PathParams;
}


export class GetUserV2Response extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  userV2?: shared.UserV2;
}
